src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-windows`
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Expected $false, but got $true.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 112
112:                 $result1[0].Options.DontFragment | Should -BeFalse
```
Test-Connection.Ping.Repeat works
```powershell
Expected 32, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 208
208:                 $pingResults.Where{ $_.Status -eq 'Success', 'Default', 1 }.BufferSize | Should -Be 32
```